crates/project/src/project.rs

```diff
@@ -217,6 +216,7 @@ enum ProjectClientState {
 /// A prompt requested by LSP server.
 #[derive(Clone, Debug)]
 pub struct LanguageServerPromptRequest {
+    pub level: PromptLevel,
```

Comment:
Don't we need to use that prompt level in workspace.rs? e.g. here: https://github.com/zed-industries/zed/pull/6988/files#diff-041673bbd1947a35d45945636c0055429dfc8b5985faf93f8a8a960c9ad31e28R585